.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+## Actual Behavior
```

Comment:
Should "Expected Behavior" come before the "Actual Behavior"?
Usually when you run into an issue, you expect X, but observe Y 😄